### ISSUE TITLE
`stellar tx new` subcommands have long difficult to read helps

### ISSUE THREAD

**Issue by leighmcculloch** [stellar/stellar-cli#1794]
### What version are you using?

22.0.1

### What did you do?

```
$ stellar tx new
```

### What did you expect to see?

Succinct one-liners for each type of operation. Easy to read and parse. Longer details only when drilling into each command, for example with `stellar tx new create-account --help`.

### What did you see instead?

<img width="989" alt="Image" src="https://github.com/user-attachments/assets/74d8fc69-a8e9-41bb-93a5-a8889fbe44da" />

**Comment by estherbreath** [comment]
Can I work on this, please?

**Comment by estherbreath** [comment]
### Can I work on this, please?

**Comment by janewang** [comment]
@estherbreath Yes feel free to pick up this one. I think the formatting could be intended where descriptions are tabbed in

**Comment by estherbreath** [comment]
@leighmcculloch could you please shed more light on this issue?
point me to where I can find the 'stellar tx new subcommand' file, please

**Comment by leighmcculloch** [comment]
I haven't worked on that area of code, but the new command code appears to be in this folder: 
`cmd/soroban-cli/src/commands/tx/new`

**Comment by estherbreath** [comment]
Alright. When should I expect the full details, please?
